src/structs/field.ts

```diff
@@ -122,30 +122,41 @@ ${this.isThreadStatic || this.isLiteral ? `` : ` // 0x${this.offset.toString(16)
         }
 
         /** @internal */
-        withHolder(instance: Il2Cpp.Object | Il2Cpp.ValueType): Il2Cpp.Field<T> {
+        withHolder(instance: Il2Cpp.Object | Il2Cpp.ValueType): Il2Cpp.HeldField<T> {
```

Comment:
This relates to https://github.com/vfsfitvnm/frida-il2cpp-bridge/pull/598/files#r1948055781.

I would rename it to `bind`, and make it public (that is removing the `/** @internal */`).